client/src/modules/dashboard/Events/components/EventForm.tsx

```diff
@@ -299,6 +299,7 @@ const EventForm: React.FC<EventFormProps> = (props) => {
               label="Streaming URL"
               placeholder=""
               isDisabled={loading}
+              isRequired
```

Comment:
An event is quite a generic thing, so it may be streamed or it may not.  It's quite possible someone could just plan to meet on Google meet, so streaming url would not be necessary.
```suggestion
```